.github/workflows/deploy_pages.yaml

```diff
@@ -10,6 +10,10 @@ on:
       - '.github/workflows/deploy_pages.yaml'
       - 'pkgs/markdown/**'
 
+defaults:
```

Comment:
We don't want this - we want to be able to deploy to pages anything from this repo that we want to host (currently just the markdown playground).